doc/bigint.md

```diff
@@ -0,0 +1,93 @@
+# BigInt
+
+A JavaScript BigInt value.
+
+## Methods
+
+### New
+
+```cpp
+static BigInt New(Napi::Env env, int64_t value);
+static BigInt New(Napi::Env env, uint64_t value);
+```
+
+ - `[in] env`: The `napi_env` Environment
+ - `[in] value`: The value the JavaScript BigInt will contain
+
+These APIs convert the C `int64_t` and `uint64_t` types to the JavaScript
+`BigInt` type.
+
+```cpp
+static BigInt New(Napi::Env env,
+                  int sign_bit,
+                  size_t word_count,
+                  const uint64_t* words);
+```
+
+ - `[in] env`: The `napi_env` Environment
```

Comment:
The parameter env is Napi::Env  and in the rest of documentation we defined this parameter as "The environment in which to ..."